blobfile/ops.py

```diff
@@ -45,8 +45,6 @@
 
 if TYPE_CHECKING:
     # only supported in python 3.8+
-    # because this isn't used for a base class or casting, we don't need to postpone

```

Comment:
I don't see what's inaccurate about this comment, could you clarify?

Reply:
Well the part about pep-536 seems wrong, but if `Literal` is used for a base class or casting, it seems like this causes python 3.7 programs to fail.

Reply:
Yeah, the inaccuracy is that to use `Literal` at all (without `typing_extensions`) in 3.7, we need `from __future__ import annotations`.

I guess I'd phrase this as:
```
This works because we postponed evaluation of type annotations with PEP 563, and because we don't use Literal as a base class or for casting
```
I'll push a commit with this wording, but let me know if it can be improved further :-)

Reply:
That's better, thanks!